### DISCUSSION TITLE
RFC: Proposal for Economic Simplification and Subitizing of the Crab Chain

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1273] [General]
## Introduction:
A network with its native token usually implies the existence of an independent crypto-economy. Darwinia, given its community constraints, may find it challenging to nurture two separate crypto-economies. The primary goal is to restructure the Crab chain's economy in a manner that it better serves the Darwinia economy, ensuring concentrated growth and effective resource utilization.

## Rationale:
The Crab chain's inception was fundamentally to act as a canary and experimental network, dedicated to serving the Darwinia chain and its Treasury. This foundational premise suggests a compelling reason for the Darwinia Treasury to consistently support the Crab chain, potentially transitioning it into a SubDAO or a subchain. Such a transformation would involve the Crab chain assisting the Darwinia chain, augmented by xRING's liquidity and financial support from the Darwinia Treasury.

### Cease CRAB Token Minting: 

Terminate the inflationary practices and minting of new CRAB Tokens. This eliminates the necessity to allocate resources for devising and overseeing an independent monetary policy.

### Token Utility: 

Maintain the CRAB token's role for governance and utility within the Subchain (Crab chain).

### Security Expenses Reallocation: 

Costs associated with security, which include securing Kusama parachain slots and Collator Staking incentives, were historically offset by inflation. These financial obligations should now fall upon the Crab chain's Treasury. Should these funds be depleted, the next step would be to solicit xRING support from the Darwinia Treasury. This entails a swapping mechanism wherein xRING, presently ensuring liquidity, would be exchanged for CRAB.

### Subsystem Prosperity: 

Effective subdaos have the potential to flourish without being resource-drained by inefficient ones. They shouldn't jeopardize the overall well-being of the Darwinia ecosystem. If any SubDAO or subchain poses such a threat, the Darwinia Treasury should reconsider its support, possibly allowing them to either seek independence or face dissolution. If circumstances dictate, CRAB Token holders have the authority, via governance voting, to incinerate their CRAB tokens and liquidate treasury assets, such as xRING. An impending challenge to ponder upon is identifying successor EVM chains that can inherit and sustain the experimental dApps presently running on the Crab chain.

## Reference:
For a deeper insight into Subdaos and their operation, please visit: [MakerDAO's Subdaos Overview](https://endgame.makerdao.com/subdaos/overview).

**Comment by boundless-forest** [comment]
Very constructive view. I totally agree that the Darwinia team should fully focus on building an excellent network. While The Crab network is  fulfilling its function as a canary network, it is also hindering the Darwinia network ecosystem building to a certain extent. This is clear if you check out the transactions active token holder amount. So I agree with this idea and think that the Darwinia network will benefit from it in the long run. I'd like to discuss something in more detail. I propose to create a new github organisation(SubDAO or something else) to split the the current crab runtime and maintain it separately from the current repo, then inform the ecosystem to attract interested members to anticipate the operation of this DAO.

**Comment by xiaoch05** [comment]
I completely agree with this constructive idea. Crab Network, as the precursor network to Darwinia Network, has been instrumental in verifying and addressing numerous issues on the Darwinia mainnet over the past few years. The engineering team and the community have limited resources, and maintaining two independent crypto ecosystems can divert their attention. Furthermore, the current liquidity of Crab tokens is insufficient to support its full independent operation. As a precursor network, I also believe it should boldly attempt some reforms to better serve the Darwinia ecosystem.

Regarding the handling of Crab tokens, I propose a suggestion. When Crab tokens become governance tokens, Crab Network's gas tokens will be swapped for xRING on the Darwinia chain, and the earnings on Crab Network will be used to repurchase Crab tokens and burn them. Since, at this point, Crab Network's operating expenses will rely on financial support from Darwinia, when the treasury is insufficient to support the network's operation, the Crab network will be shut down, and a certain exchange rate will be set on the Darwinia network. The treasury will fund the conversion of Crab tokens into RING.

**Reply by SasoLithops** [reply]
Do not forget many of us hold Ckton and have Crab locked for many years. May I know in simple terms what is going on with Crab chain now and what is the plan to inform community.

**Comment by hackfisher** [comment]
https://medium.com/darwinianetwork/crab-6501-runtime-upgrade-39722670a3c2
